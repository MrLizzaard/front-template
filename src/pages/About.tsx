import React from "react";

const About = () => {
  return <h2 role="banner">About</h2>;
};

export default About;
