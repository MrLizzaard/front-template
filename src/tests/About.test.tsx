import { render, screen } from "@testing-library/react";
import About from "../pages/About";

describe("About 테스트", () => {
  test("스냅샷", () => {
    // render();
    const utils = render(<About />);
    expect(utils.container).toMatchSnapshot();
  });

  it("h2 test", () => {
    render(<About />);

    const h2 = screen.getByRole("banner");
    expect(h2?.innerHTML).toBe("About");
  });
});
