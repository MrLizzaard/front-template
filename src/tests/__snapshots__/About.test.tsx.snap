// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About 테스트 스냅샷 1`] = `
<div>
  <h2
    role="banner"
  >
    About
  </h2>
</div>
`;
